package.json

```diff
@@ -5,7 +5,10 @@
   "main": "src/index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "lint": "eslint . --ext '.js' --cache"
+    "lint": "eslint . --ext '.js' --cache",
+    "build:dev": "webpack --mode development",
+    "build:prod": "webpack --mode production",
+    "postinstall": "webpack --mode production"
```

Comment:
Why do we need this `postinstall` script?

Reply:
Removed